xml/System.Numerics/Vector.xml

```diff
@@ -922,7 +922,7 @@
       </Parameters>
       <Docs>
         <param name="value">The source vector.</param>
-        <summary>Converts a <see cref="T:System.Numerics.Vector{System.Int64}" /> to a <see cref="T:System.Numerics.Vector{System.Double}" />.</summary>
+        <summary>Converts a <see cref="T:System.Numerics.Vector" />&lt;<see cref="T:System.Int64" />&gt; to a <see cref="T:System.Numerics.Vector" />&lt;<see cref="T:System.Double" />&gt;.</summary>
```

Comment:
These changes are somewhat wrong since the link then goes to Vector instead of Vector\<T>. We might have just to do this instead since I checked and VS does nothing special with that formatting:

```suggestion
        <summary>Converts a <c>System.Numerics.Vector&lt;Int64&gt;</c> to a <c>System.Numerics.Vector&lt;Double&gt;</c>.</summary>
```